Categories/UIView+TCAdditions.m

```diff
@@ -129,5 +129,19 @@ - (UIView *)tc_breadthFirstSubviewThatSatisfiesBlock:(BOOL (^)(UIView *))conditi
     return nil;
 }
 
+- (void)tc_autoLayoutPinToParentViewController {
```

Comment:
Yea, I think at very least the name should be changed. There are also a number of great AutoLayout libraries out there that cover this ground. @nhgrif What do you think? Do you still want to include it?